src/components/CategorySearchFilter.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Chip } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ResultParam, TicketSearchParam } from '../api/searchApi';
+import { CategoryFilterDialog } from '../pages/search/advanced_search/CategoryFilterDialog';
+import { PublicationInstanceType } from '../types/registration.types';
+import { CategoryChip } from './CategorySelector';
+
+interface CategorySearchFilterProps {
+  searchParam: ResultParam | TicketSearchParam;
```

Comment:
Mulig dette skulle vært enda mer eksplisitt?

Reply:
Godt poeng! Det er vel i praksis bare disse atm?

```suggestion
  searchParam: ResultParam.CategoryShould | TicketSearchParam.PublicationType;
```